packages/backend/src/functions/oauth/GitHub.ts

```diff
@@ -99,11 +100,22 @@ export default class GitHub {
 
     const user = await this.getUser(userId);
     if (isUser(user)) {
-      updateUser(filterUser(user));
+      if (await isFirstTime()) {
+        updateUser(filterUser(user));
+        unsetFirstTime();
+      } else if (await existUser(user)) {
+        /**user exist so do nothing */
+        // eslint-disable-next-line no-constant-condition
+      } else if (false /* is in pre-approved list?*/) {
```

Comment:
This too

Reply:
It's for upcoming issue. I wanna keep it